README.md

```diff
@@ -256,6 +256,12 @@ stub_request(:any, "www.example.com").
 Net::HTTP.get("www.example.com", '/')    # ===> "abc"
 ```
 
+Set appropriate Content-Type for HTTParty's `parsed_response`.
+
+```ruby
+stub_request(:any, "www.example.com").to_return body: '{}', headers: {content_type: 'application/json'}
+```
+
```

Comment:
Don't know when this left the readme, but I had to go on quite the hunt to figure out I needed it. Where did this tasty little tidbit get moved to?